### PR TITLE
Split build() for readability

### PR DESCRIPTION
**Pull request by Geobert**
I've split cobalt.rs build() function to make it easier to read

### CODE REVIEW

**Comment by Geobert** [comment]
I'll do the renaming tonight :)

File: src/cobalt.rs [hunk 1]
```diff
@@ -173,89 +205,121 @@ pub fn build(config: &Config) -> Result<()> {
...
-    if let Some(ref path) = config.posts.rss {
-        create_rss(path, dest, config, &posts)?;
-    }
-    // check if we should create an jsonfeed file and create it!
-    if let Some(ref path) = config.posts.jsonfeed {
-        create_jsonfeed(path, dest, config, &posts)?;
+    Ok(())
+}
+
+fn sort_documents_by_date(posts: &mut Vec<Document>, config: &Config) -> Result<()> {
```
[the remark this answers is not in the document]
Comment (Geobert) [reply src/cobalt.rs]: So, what should I call it, as the code is really doing sorting by date ^^'

[the remark this answers is not in the document]
Comment (epage) [reply src/cobalt.rs]: `sort_pages`?

[the remark this answers is not in the document]
Comment (Geobert) [reply src/cobalt.rs]: Oh! I misread, I though you wanted me to drop the "sort" word. You were talking "sort *policy*" ^^'

[the remark this answers is not in the document]
Comment (epage) [reply src/cobalt.rs]: I can see how that can be missed and be confusing :)

**Pull request merged**